src/runtime/print.go

```diff
@@ -201,6 +201,8 @@ func printitf(msg interface{}) {
 	switch msg := msg.(type) {
 	case string:
 		print(msg)
+	case error:
+		print(msg.Error())
 	default:
```

Comment:
Thought, do we want to add a `case fmt.Stringer`?

Reply:
Good idea, `panic()` in the Go runtime also seems to accept that: https://play.golang.org/p/UOeakJdGX6_3